paasta_tools/contrib/get_running_task_allocation.py

```diff
@@ -122,14 +122,18 @@ def get_kubernetes_resource_request(
 ) -> Mapping[str, float]:
     if not resources:
         requests: Mapping[str, str] = {}
+        limits: Mapping[str, str] = {}
```

Comment:
super-duper-nit/feel free to ignore me: iirc, Python convention is to use more concrete types (e.g., `Dict`) for return value types and leave more abstract types/interfaces (e.g., `Mapping`) for inputs

for example, imagine a function that returns a `collections.Counter`: telling the type-checker that we're returning a Mapping means that it can't help us later on if we actually want to use some of the convenience methods that are added onto a Counter since, as far as mypy knows, we've returned something that only supports basic dictionary functionality

we've been kinda inconsistent about this tho, so feel free to ignore me :)